code/style.css

```diff
@@ -5,18 +5,102 @@
 body {
   margin: 0;
   padding: 0;
-  font-family: 'Montserrat', sans-serif;
-  background: #0026ff;
+  /* background-image: url(./assets/background.jpeg); */
+  background-color: #585858;
+  font-family: 'Palette Mosaic', cursive;
```

Comment:
Perfect font for a sushi place! And the colors as well!

Reply:
We were trying to find a font that is more Japanese, but couldn't find it. Some really nice fonts costs so we used that one int the project. Do you have some ideas for fonts? :)

Reply:
This site is good - it has true type fonts. I think you can add it?

https://www.dafont.com/search.php?q=japanese

Reply:
Thanks for tip! :)